content/en/docs/setup/minikube.md

```diff
@@ -192,48 +249,42 @@ minikube start \
     --extra-config=kubelet.image-service-endpoint=/var/run/crio.sock \
     --bootstrapper=kubeadm
 ```
-
-#### rkt container engine
-
+{{% /tab %}}
+{{% tab name="rkt container engine" %}}
 To use [rkt](https://github.com/rkt/rkt) as the container runtime run:
-
 ```shell
 minikube start \
     --network-plugin=cni \
     --enable-default-cni \
     --container-runtime=rkt
 ```
-
 This will use an alternative minikube ISO image containing both rkt, and Docker, and enable CNI networking.
+{{% /tab %}}
+{{< /tabs >}}
 
-### Driver plugins
-
-See [DRIVERS](https://git.k8s.io/minikube/docs/drivers.md) for details on supported drivers and how to install
-plugins, if required.
-
-### Use local images by re-using the Docker daemon
+#### Use local images by re-using the Docker daemon
 
-When using a single VM of Kubernetes, it's really handy to reuse the Minikube's built-in Docker daemon; as this means you don't have to build a docker registry on your host machine and push the image into it - you can just build inside the same docker daemon as minikube which speeds up local experiments. Just make sure you tag your Docker image with something other than 'latest' and use that tag while you pull the image. Otherwise, if you do not specify version of your image, it will be assumed as `:latest`, with pull image policy of `Always` correspondingly, which may eventually result in `ErrImagePull` as you may not have any versions of your Docker image out there in the default docker registry (usually DockerHub) yet.
+When using a single VM for Kubernetes, it's useful to reuse Minikube's built-in Docker daemon. This means you don't have to build a Docker registry on your host machine and push the image into it. Instead, you can build a Docker daemon inside Minikube which speeds up local experiments. Just make sure you tag your Docker image with something other than 'latest' and use that tag while you pull the image. If you do not specify the version of your image, it is assumed as `:latest` and if the image pull policy is set to  `Always`, this eventually results in an `ErrImagePull` as you may not have the Docker image in the default Docker registry (usually DockerHub). Therefore, remember to turn off the `imagePullPolicy:Always` because it would not allow Kubernetes to use the images you built locally.
 
-To be able to work with the docker daemon on your mac/linux host use the `docker-env command` in your shell:
+To be able to work with the Docker daemon on your Mac/Linux host, use the `docker-env command` in your shell:
 
 ```shell
 eval $(minikube docker-env)
 ```
 
-You should now be able to use docker on the command line on your host mac/linux machine talking to the docker daemon inside the minikube VM:
+You are now able to use Docker on the command line of your host Mac/Linux machine to talk to the Docker daemon inside the Minikube VM:
 
 ```shell
 docker ps
 ```
-
-On Centos 7, docker may report the following error:
+{{< note >}}
```

Comment:
Be sure to close this note with the corresponding `{{< /note >}}` tag.

Reply:
Noted 👍 . I have closed it below (the note is a bit long due to the code snippet that is added).